model/Licensing/Licensing.md

```diff
@@ -45,11 +45,11 @@ types of Software Artifacts. For example:
     different File (e.g., comment at top of File if it is not within the
     Snippet, LICENSE file in the top directory of a repository)
 
-A declaredLicense is related to a NoneLicenseindicates that the
+A declaredLicense is related to a NoneLicense and indicates that the
 corresponding Package, File or Snippet contains no license information
 whatsoever.
 
-A declaredLicense related to aNoAssertionLicense
+A declaredLicense related to a NoAssertionLicense and 
```

Comment:
Same thing here, "A declaredLicense relationship to a NoAssertionLicense indicates..."